ibis/omniscidb/ddl.py

```diff
@@ -120,21 +120,29 @@ def compile(self):
 class CreateTableWithSchema(CreateTable):
     """Create Table With Schema class."""
 
-    def __init__(self, table_name, schema, database=None, max_rows=None):
+    def __init__(
+        self,
+        table_name,
+        schema,
+        database=None,
+        max_rows=None,
+        fragment_size=None,
+    ):
         self.table_name = table_name
         self.database = database
         self.schema = schema
         self.max_rows = max_rows
+        self.fragment_size = fragment_size
 
     @property
     def with_params(self):
         """Return the parameters for `with` clause.
```

Comment:
can you type return values

Reply:
Return value type is below, 145 line

Reply:
@semelianova you should do something like that:
```with_params(self) -> [str or something else]```

Reply:
return value is Dict[str, Any]